core/bug_api.php

```diff
@@ -254,6 +264,11 @@ class BugData {
 	private $loading = false;
 
 	/**
+	 * Indicates if create() has used existed bug or created new bug
+	 */
+	private $aliased = false;
```

Comment:
This doesn't belong on the bug information.  This is specific the report issue API and should be part of its response only.